src/components/chart/_test-examples.js

```diff
@@ -46,6 +46,47 @@ export const EXAMPLE_LINE_CHART_WITH_CONFIG_PARAMS = {
     percentageHeightMobile: 120,
 };
 
+export const EXAMPLE_LINE_CHART_WITH_LEGEND_TRUE_PARAMS = {
+    chartType: 'line',
+    title: 'Example Line Chart with legend set to be true',
+    subtitle: 'A sample subtitle',
+    id: 'line-chart-legend-true-123',
+    legend: true,
+    xAxis: { title: 'X Axis Title', categories: ['Jan', 'Feb', 'Mar'] },
+    yAxis: { title: 'Y Axis Title' },
+    series: [
+        { name: 'Series 1', data: [10, 20, 30] },
+        { name: 'Series 2', data: [15, 25, 35] },
+    ],
+};
+
+export const EXAMPLE_LINE_CHART_WITH_LEGEND_FALSE_PARAMS = {
+    chartType: 'line',
+    title: 'Example Line Chart with legend set to be false',
+    subtitle: 'A sample subtitle',
+    id: 'line-chart-legend-false-123',
+    legend: false,
+    xAxis: { title: 'X Axis Title', categories: ['Jan', 'Feb', 'Mar'] },
+    yAxis: { title: 'Y Axis Title' },
+    series: [
+        { name: 'Series 1', data: [10, 20, 30] },
+        { name: 'Series 2', data: [15, 25, 35] },
+    ],
+};
+
+export const EXAMPLE_LINE_CHART_WITH_LEGEND_UNSET_PARAMS = {
+    chartType: 'line',
+    title: 'Example Line Chart with legend unset',
+    subtitle: 'A sample subtitle',
+    id: 'line-chart-legend-unset-123',
+    xAxis: { title: 'X Axis Title', categories: ['Jan', 'Feb', 'Mar'] },
+    yAxis: { title: 'Y Axis Title' },
+    series: [
+        { name: 'Series 1', data: [10, 20, 30] },
+        { name: 'Series 2', data: [15, 25, 35] },
+    ],
+};
```

Comment:
Theres a lot of repitition here, you could just have one example with the same config as "EXAMPLE_LINE_CHART_WITH_LEGEND_UNSET_PARAMS" then in the test just use the same example each time and add the legend param setting in the test itself. Its a pattern we have used elsewhere

Reply:
This should now be addressed.